test/general-spec.js

```diff
@@ -81,8 +82,50 @@ describe('Functionality that is common to all documents:', function() {
       expect(syncFunction).withArgs(doc).to.throwException(function(ex) {
         expect(ex.message).to.equal(expectedError.message);
       });
-      expect(requireAccess.callCount).to.equal(1);
-      expect(requireAccess.calls[0].arg).to.eql('remove');
+      expect(requireAccess.callCount).to.be(1);
+      expect(requireAccess.calls[0].arg.length).to.be(2);
+      expect(requireAccess.calls[0].arg).to.contain('delete');
+      expect(requireAccess.calls[0].arg).to.contain('remove');
+    });
+  });
+
+  describe('channel assignment', function() {
```

Comment:
Verifies the channels that are assigned to a document via the `channel` function, which are a superset of the channels that are required to perform each operation (i.e. that are passed to the `requireAccess` function).